src/Proto.Persistence/InMemoryProviderState.cs

```diff
@@ -60,5 +51,15 @@ public Task PersistSnapshotAsync(string actorName, ulong eventIndex, object snap
             _snapshots[actorName] = Tuple.Create((object) snapshot, eventIndex);
             return Task.FromResult(0);
         }
+
+        public Task DeleteEventsAsync(string actorName, ulong fromEventIndex)
+        {
+            throw new NotImplementedException();
+        }
+
+        public Task DeleteSnapshotsAsync(string actorName, ulong fromEventIndex)
+        {
+            throw new NotImplementedException();

```

Comment:
These should probably be implemented as well. If we want to keep this class - I'm not sure if it's needed for anything except testing (in which case it should be moved)